src/components/GithubPopularRepositories.tsx

```diff
@@ -23,8 +23,9 @@ const GithubPopularRepositories: FC<Props> = ({ repositories }) => {
           md: 'repeat(2, 1fr)',
         }}
       >
-        {repositories.map((repository) => (
-          <Link
+        {repositories.map((repository) => {
+          if(!Object.keys(repository).length) return <></>
```

Comment:
This is to avoid negative conditions and performant checks.

```suggestion
          if(Object.entries(repository).length === 0) return <></>
```

Reply:
Fixed